### DISCUSSION TITLE
Is there any legit source where I get a pre-patched apk ?

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#283] [Question] [answered]
Not telegram. Any other valid sources? I just don't want to compile it myself due to the lack of server.

**Comment by oSumAtrIX** [comment, marked as answer]
No. ReVanced does not share any pre-patched application.

**Reply by ghost** [reply]
Tq for a quick reply.
Is there any way to get telegram group's APK without a telegram account?

**Reply by oSumAtrIX** [reply]
We do not share any APK through Telegram.

**Comment by ghost** [comment]
Just wait manager to be publicly released.

**Reply by ghost** [reply]
I guess the manager would be a patching frontend rather than getting a pre-patched app from a random server right? (Probably with all cli toggles). It would be great if that's the case.

**Reply by TheJeterLP** [reply]
> I guess the manager would be a patching frontend rather than getting a pre-patched app from a random server right? (Probably with all cli toggles). It would be great if that's the case.

Exactly. There is no prepatched APK from a legit source and will never be

**Comment by bropines** [comment]
> Not telegram. Any other valid sources? I just don't want to compile it myself due to the lack of server.

4pda. but everything is in Russian there)

**Comment by Jarvis-Ank** [comment]
> Not telegram. Any other valid sources? I just don't want to compile it myself due to the lack of server.

You can build it via Chrome.

**Reply by ghost** [reply]
Ofc but atleast need to get a throwaway server. And a bit of sparetime, which I really lack. 
I wish to try building it on my personal laptop like on sundays though.

**Reply by Jarvis-Ank** [reply]
I've built a script just for that, it works on VM (Google Colab) doesn't require you to do anything other than uploading latest YouTube and running the script. Google Colab is free too.

**Reply by ghost** [reply]
> I've built a script just for that, it works on VM (Google Colab) doesn't require you to do anything other than uploading latest YouTube and running the script. Google Colab is free too.

Lmao, using google's services to patch google's application. It became it's own villan it swore to destroy. xD

**Reply by Jarvis-Ank** [reply]
Lol.. btw it works on mobile/pc

**Reply by bropines** [reply]
> Lol.. btw it works on mobile/pc

hmmm. Will it be possible to assemble on termux?

**Reply by TheJeterLP** [reply]
> > Lol.. btw it works on mobile/pc
> 
> hmmm. Will it be possible to assemble on termux?

No.

**Comment by TheJeterLP** [comment]
> Not telegram. Any other valid sources? I just don't want to compile it myself due to the lack of server.

No server needed.
